src/docs/components/styling.md

```diff
@@ -5,165 +5,207 @@ url: /docs/styling
 
-In the example above, since the namespace is `app`, the generated global styles file will be located at: `./www/build/app.css`.
+### Scoped CSS
 
-This file must be manually imported in the `index.html` of your application, which you can find in `src/index.html`:
+An alternative to using the shadow DOM is using scoped components. You can use scoped components by setting the `scoped` option to `true` in the component decorator.
 
 ```tsx
-<link rel="stylesheet" href="/build/app.css">
+@Component({
+  tag: 'shadow-component',
+  styleUrl: 'shadow-component.css',
+  scoped: true,
+})
+export class ShadowComponent {}
 ```
 
-Keep in mind that global styles should be reserved for **global** styles, ie, you should try to avoid styling your components with it, instead, some interesting use cases can be:
-
-- Theming: defining CSS variables used across the app
-- Load fonts with `@font-face`
-- App wide font-family
-- Style body background
-- CSS resets
+Scoped CSS is a proxy for style encapsulation. It works by appending a data attribute to your styles to make them unique and thereby scope them to your component. It does not, however, prevent styles from the light DOM from seeping into your component.
 
+In browsers that do not currently support Shadow DOM, web components built with Stencil will fall back to using scoped CSS instead of loading a large Shadow DOM polyfill.
 
-## CSS Variables
+## CSS Custom Properties
 
-### What are CSS Variables?
+CSS custom properties, also often referred to as CSS variables, are used to contain values that can then be used in multiple CSS declarations. For example, we can create a custom property called `--color-primary` and assign it a value of `blue`.
 
-[CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables) are a lot like [Sass Variables](https://ionicframework.com/docs/theming/sass-variables/), but built into the browser. CSS Variables allow you to specify CSS properties that can be used across your app.
+```css
+:host {
+  --color-primary: blue;
+}
+```
 
-### Use Case
+And then we can use that custom property to style different parts of our component
 
-One use case for CSS Variables is colors. If your app has a primary brand color that is used across your app then instead of writing that same color out each place you need it in your app you can create a variable for it and then use that variable anywhere you need that color in your app. Also, if you ever need to change this color you will only have to change the variable and then it will be updated across your app.
+```css
+h1 {
+  color: var(--color-primary);
+}
+```
 
-### Using CSS Variables in Stencil
+### Customizing Components with Custom Properties
 
-Here are the recommended steps to use CSS Variables in Stencil:
+CSS custom properties can allow users of a component to customize a component’s styles from the light DOM. Consider a `shadow-card` component that uses a custom property for the color of the card heading.
```

Comment:
I'm not sure I fully understand this sentence

> CSS custom properties can allow users of a component to customize a component’s styles from the light DOM.

I can do something like 
```
:host {
  --heading-color: blue;
}

div {
  background: var(--heading-color);
}
```
in a CSS file that is only used in a component that uses the shadow DOM.  My guess is I need more coffee 😆

Reply:
I'm not sure I totally understand, but what I'm trying to demonstrate here is that if you declare a custom property on the host, that custom property can be changed by the consumer of the component to customize the component's styles

Reply:
Ah, gotcha. That makes sense to me.  Maybe incorporating that into the first sentence here would help make it a little clearer:

```suggestion
CSS custom properties can allow the consumers of a component to customize a component’s styles from the light DOM. Consider a `shadow-card` component that uses a custom property for the color of the card heading.
```